samples/chat/csharp-ooproc/ChatBotIsolated.cs

```diff
@@ -72,7 +72,7 @@ public class CreateChatBotOutput
 
     public class PostResponseOutput
     {
-        [ChatBotPostOutput("{chatId}")]
```

Comment:
I can't suggest changes directly to the line so here's the code change example for CreateChatBot:

```
[Function(nameof(CreateChatBot))]
    public static async Task<CreateChatBotOutput> CreateChatBot(
            [HttpTrigger(AuthorizationLevel.Anonymous, "put", Route = "chats/{chatId}")] HttpRequestData req,
            string chatId)
    {
        var responseJson = new { chatId = chatId };

        using StreamReader reader = new StreamReader(req.Body);
        
        string request = await reader.ReadToEndAsync();

        CreateRequest? createRequestBody = JsonSerializer.Deserialize<CreateRequest>(request);

        if (createRequestBody == null)
        {
            throw new ArgumentException("Invalid request body. Make sure that you pass in {\"instructions\": value } as the request body.");
        }

HttpResponseData response = req.CreateResponse(HttpStatusCode.Created);
await response.WriteAsJsonAsync(responseJson);

        return new CreateChatBotOutput
        {
            HttpResponse =response,
            ChatBotCreateRequest = new ChatBotCreateRequest(chatId, createRequestBody.Instructions),
        };
    }

    public class CreateChatBotOutput
    {
        [ChatBotCreateOutput()]
        public ChatBotCreateRequest? ChatBotCreateRequest { get; set; }

        public HttpResponseData? HttpResponse { get; set; }
    }
```

Reply:
This would ensure that we return 201 and we also return the body

Reply:
This is the code change for `PostUserResponse`:
```
[Function(nameof(PostUserResponse))]
    public static async Task<PostResponseOutput> PostUserResponse(
        [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "chats/{chatId}")] HttpRequestData req,
        string chatId)
    {
        string? userMessage = await req.ReadAsStringAsync();

        if (string.IsNullOrEmpty(userMessage))
        {
            HttpResponseData badResponse = req.CreateResponse(HttpStatusCode.BadRequest);
            await badResponse.WriteStringAsync("Request body is empty");
            return new PostResponseOutput { HttpResponse = badResponse };
        }

        HttpResponseData response = req.CreateResponse(HttpStatusCode.Accepted);

        return new PostResponseOutput
        {
            HttpResponse = response,
            ChatBotPostRequest = new ChatBotPostRequest { UserMessage = userMessage, Id = chatId }
        };
    }

    public class PostResponseOutput
    {
        [ChatBotPostOutput("{chatId}")]
        public ChatBotPostRequest? ChatBotPostRequest { get; set; }

        public HttpResponseData? HttpResponse { get; set; }
    }
````